diff --git a/node_modules/@rpearce/image-zoom/dist/esm/index.js b/node_modules/@rpearce/image-zoom/dist/esm/index.js
index 987d491..888f3ed 100644
--- a/node_modules/@rpearce/image-zoom/dist/esm/index.js
+++ b/node_modules/@rpearce/image-zoom/dist/esm/index.js
@@ -55,7 +55,7 @@ var WIDTH = 'width';
 var ZERO = '0';
 var Z_INDEX_CSS = 'z-index';
 var ImageZoom = function (_a, targetEl) {
-    var _b = _a === void 0 ? {} : _a, _c = _b.closeText, closeText = _c === void 0 ? 'Unzoom image' : _c, _d = _b.isControlled, isControlled = _d === void 0 ? false : _d, _e = _b.modalText, modalText = _e === void 0 ? 'Zoomed item' : _e, onZoomChange = _b.onZoomChange, _f = _b.openText, openText = _f === void 0 ? 'Zoom image' : _f, _g = _b.overlayBgColor, overlayBgColor = _g === void 0 ? '#fff' : _g, _h = _b.overlayOpacity, overlayOpacity = _h === void 0 ? 0.95 : _h, _j = _b.transitionDuration, _transitionDuration = _j === void 0 ? 300 : _j, _k = _b.zoomMargin, zoomMargin = _k === void 0 ? 0 : _k, _l = _b.zoomZindex, zoomZindex = _l === void 0 ? 2147483647 : _l;
+    var _b = _a === void 0 ? {} : _a, _c = _b.closeText, closeText = _c === void 0 ? 'Unzoom image' : _c, _d = _b.isControlled, isControlled = _d === void 0 ? false : _d, _e = _b.modalText, modalText = _e === void 0 ? 'Zoomed item' : _e, onZoomChange = _b.onZoomChange, _f = _b.openText, openText = _f === void 0 ? 'Zoom image' : _f, _g = _b.overlayBgColor, overlayBgColor = _g === void 0 ? '#fff' : _g, _h = _b.overlayOpacity, overlayOpacity = _h === void 0 ? 1 : _h, _j = _b.transitionDuration, _transitionDuration = _j === void 0 ? 300 : _j, _k = _b.zoomMargin, zoomMargin = _k === void 0 ? 0 : _k, _l = _b.zoomZindex, zoomZindex = _l === void 0 ? 2147483647 : _l;
     var isImgEl = targetEl.tagName === 'IMG';
     var isSvgSrc = isImgEl && SVG_REGEX.test(targetEl.currentSrc);
     var isImg = !isSvgSrc && isImgEl;
@@ -131,10 +131,12 @@ var ImageZoom = function (_a, targetEl) {
             : getScaleToWindow(width, height, zoomMargin);
         if (currentScale > 1) {
             // create openBtnEl
-            openBtnEl = createElement(BUTTON);
-            setAttribute(ARIA_LABEL, openText, openBtnEl);
+            openBtnEl = createElement(DIV);
+            setAttribute('aria-hidden', 'true', openBtnEl);
+            setAttribute('tabindex', '-1', openBtnEl);
             setAttribute(STYLE, styleZoomBtnIn, openBtnEl);
-            setAttribute(TYPE, BUTTON, openBtnEl);
+            setAttribute('data-rmiz-trigger', 'true', openBtnEl)
+            // setAttribute(TYPE, BUTTON, openBtnEl);
             adjustOpenBtnEl();
             addEventListener(CLICK, handleOpenBtnClick, openBtnEl);
             // insert openBtnEl after targetEl
